src/schemathesis/commands.py

```diff
@@ -36,8 +49,27 @@ def run(schema: str, checks: Iterable[str] = DEFAULT_CHECKS_NAMES) -> None:
 
     selected_checks = tuple(check for check in runner.DEFAULT_CHECKS if check.__name__ in checks)
 
+    auth = prepare_auth(raw_auth)
+
+    headers = prepare_headers(header)
+
     click.echo("Running schemathesis test cases ...")
 
-    runner.execute(schema, checks=selected_checks)
+    runner.execute(schema, checks=selected_checks, auth=auth, headers=headers)
 
     click.echo("Done.")
+
+
+def prepare_auth(raw_auth: Optional[str]) -> Optional[Tuple[str, str]]:
+    if raw_auth is not None:
+        auth = tuple(raw_auth.split(":"))
+        return cast(Tuple[str, str], auth)
+    return None
+
+
+def prepare_headers(header: Tuple[str, ...]) -> Dict[str, str]:
+    headers = {}
+    for head in header:
+        key, value = head.split(":")
+        headers[key] = value
```

Comment:
Maybe `value.rstrip()` to remove the space between `Key: Value`?

Reply:
Hm, then it should be `lstrip()`, right? I am thinking if space and other not printable chars could be in the header value or not, so we will not remove the part of the value, what do you think?

Reply:
Yep, `lstrip`. This is what `curl` does. It removes all white-space characters at the beginning.

So doing:

```
curl http://localhost:5000 -H "Authorization:           Bearer 123   "
```

results in header value for key `Authorization`: `Bearer 123   `

Reply:
Cool! I didn't know it :) implemented. Thanks for suggesting it :)